jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx

```diff
@@ -0,0 +1,175 @@
+        >
+          Create Pipeline
+        </button>
+      </div>
+
+      <span
+        className="pachyderm-pipeline-error"
+        data-testid="Pipeline__errorMessage"
+      >
+        {errorMessage}
+      </span>
+
+      <div className="pachyderm-pipeline-input-wrapper">
+        <label
+          className="pachyderm-pipeline-input-label"
+          htmlFor="pipelineName"
+        >
+          *Name:{'  '}
+        </label>
+        <input
+          className="pachyderm-pipeline-input"
+          data-testid="Pipeline__inputPipelineName"
+          name="pipelineName"
+          value={pipelineName}
+          onChange={(e: any) => {
+            setPipelineName(e.target.value);
+          }}
+          disabled={loading}
+        ></input>
+      </div>
+      <div className="pachyderm-pipeline-input-wrapper">
+        <label className="pachyderm-pipeline-input-label" htmlFor="imageName">
+          *Image:{'  '}
+        </label>
+        <input
+          className="pachyderm-pipeline-input"
+          data-testid="Pipeline__inputImageName"
+          name="imageName"
+          value={imageName}
+          onChange={(e: any) => {
+            setImageName(e.target.value);
+          }}
+          disabled={loading}
+        ></input>
+      </div>
+      <div className="pachyderm-pipeline-input-wrapper">
+        <label
+          className="pachyderm-pipeline-input-label"
+          htmlFor="requirements"
+        >
+          Requirements:{'  '}
+        </label>
+        <input
+          className="pachyderm-pipeline-input"
+          data-testid="Pipeline__inputRequirements"
+          name="requirements"
+          value={requirements}
+          onChange={(e: any) => {
+            setRequirements(e.target.value);
+          }}
+          disabled={loading}
+          placeholder={placeholderRequirements}
+        ></input>
+      </div>
+      <div className="pachyderm-pipeline-textarea-wrapper">
+        <label
+          className="pachyderm-pipeline-textarea-label"
+          htmlFor="inputSpec"
+        >
+          Input Spec
+        </label>
+        <textarea
+          className="pachyderm-pipeline-textarea pachyderm-input"
+          data-testid="Pipeline__inputSpecInput"
+          name="inputSpec"
+          value={inputSpec}
+          onChange={(e: any) => {
+            setInputSpec(e.target.value);
+          }}
+          disabled={loading}
+          placeholder={placeholderInputSpec}
+        ></textarea>
+      </div>
+
+      <div className="pachyderm-pipeline-spec-preview pachyderm-pipeline-textarea-wrapper">
+        <label className="pachyderm-pipeline-preview-label">
+          Pipeline Spec Preview: {'  '}
+        </label>
+        <textarea
+          className="pachyderm-pipeline-spec-preview-textarea"
+          style={{backgroundColor: '#80808080'}}
+          data-testid="Pipeline__specPreview"
+          name="specPreview"
+          value={`name: ${pipelineName}
```

Comment:
Are we not going to add the image and command to the pipeline spec preview?

Reply:
I don't think we need to show the user the command, since it will not be useful to them, but I'll add the image here.